kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -2030,6 +2030,22 @@ object KyuubiConf {
       .intConf
       .createOptional
 
+  val SERVER_LIMIT_CONNECTIONS_CUSTOM_ENABLE: ConfigEntry[Boolean] =
+    buildConf("kyuubi.server.limit.connections.custom.enable")
+      .doc("Whether to enable custom limit. If it is enabled, Kyuubi will limit the" +
+        " conn's num according to user")
+      .version("1.6.1")
```

Comment:
the next feature version is 1.7.0

Reply:
I have makd some adjust and commited according your suggestion

Reply:
I have maked some adjust and committed it according to your suggestion